### PR TITLE
Updates courage to @okta/courage@4.6.0-beta.2702.g116a94b

### PR DESCRIPTION
**Pull request by bretterer**
Minor update to include the work in courage for setting focus to first error field.

https://github.com/okta/okta-ui/pull/1395

Resolves OKTA-242468

### CODE REVIEW

File: packages/@okta/courage-for-signin-widget/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@okta/courage": "4.6.0-beta.2670.g173eed7",
+    "@okta/courage": "^4.6.0-beta.2702.g116a94b",
```
Comment (haishengwu-okta) [inline packages/@okta/courage-for-signin-widget/package.json]: probably want to be exact version `4.6.0-beta.2702.g116a94b`?

Comment (bretterer) [reply packages/@okta/courage-for-signin-widget/package.json]: Ah, didn't see that `^`....  I just ran the instructions on the readme for it.  ill remove that

Comment (haishengwu-okta) [reply packages/@okta/courage-for-signin-widget/package.json]: interesting.. usually `yarn upgrade` will pin to specific version..
btw, the yarn.lock wasn't updated.

File: packages/@okta/courage-dist/okta.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/*! THIS FILE IS GENERATED FROM PACKAGE @okta/courage@4.6.0-beta.2670.g173eed7 */
+/*! THIS FILE IS GENERATED FROM PACKAGE @okta/courage@^4.6.0-beta.2702.g116a94b */
```
Comment (manueltanzi-okta) [inline packages/@okta/courage-dist/okta.js]: I think should be exact version

File: packages/@okta/courage-for-signin-widget/yarn.lock [hunk 3]
```diff
@@ -2,9 +2,9 @@
 # yarn lockfile v1
 
 
-"@okta/courage@4.6.0-beta.2670.g173eed7":
-  version "4.6.0-beta.2670.g173eed7"
-  resolved "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/courage/-/@okta/courage-4.6.0-beta.2670+173eed7.tgz#a6fdf4a89699229cc5f12fe0947dbdcc50dcb5de"
+"@okta/courage@^4.6.0-beta.2702.g116a94b":
```
Comment (manueltanzi-okta) [inline packages/@okta/courage-for-signin-widget/yarn.lock]: Also here, I think should be exact version

**Pull request merged**